.vscode/tasks.json

```diff
@@ -0,0 +1,45 @@
+{
```

Comment:
Rather provide this in separate MR, it's not connected with rest work in this MR